dev/tasks/python-wheels/github.osx.yml

```diff
@@ -30,14 +30,26 @@ env:
   VCPKG_DEFAULT_TRIPLET: x64-osx-static-release
   VCPKG_FEATURE_FLAGS: "-manifests"
   VCPKG_OVERLAY_TRIPLETS: {{ "${{ github.workspace }}/arrow/ci/vcpkg" }}
+  # $(xcode-select -p)/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
+  # $(xcode-select -p)/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++
+  CC: "/Applications/Xcode_12.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang"
+  CXX: "/Applications/Xcode_12.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++"
 
 jobs:
   build:
     name: Build wheel for OS X
-    runs-on: macos-latest
+    runs-on: macos-10.15
     steps:
       {{ macros.github_checkout_arrow()|indent }}
 
+      - name: Install macOS SDK {{ macosx_deployment_target }}
+        run: arrow/ci/scripts/install_osx_sdk.sh
+
+      - name: Currently set SDK
```

Comment:
Seems like there is the old SDK https://github.com/ursacomputing/crossbow/runs/2898078973#step:5:1

Reply:
Still both isysroot paths get passed.